api/pytest.ini

```diff
@@ -5,5 +5,21 @@ pythonpath = .
 
 filterwarnings=
   ignore:Unverified HTTPS request
+# Ignore warnings related to unverified HTTPS requests.
```

Comment:
I'm not actually sure this suppressed warning is still needed, maybe @dhruvkb can comment since it looks like he added it initially? There's also [this somewhat related issue](https://github.com/WordPress/openverse/issues/738).